docs/api/client-request.md

```diff
@@ -49,6 +51,13 @@ following properties:
     [`request.followRedirect`](#requestfollowredirect) is invoked synchronously
     during the [`redirect`](#event-redirect) event.  Defaults to `follow`.
   * `origin` string (optional) - The origin URL of the request.
+  * `referrerPolicy` string (optional) - can be `""`, `no-referrer`,
+    `no-referrer-when-downgrade`, `origin`, `origin-when-cross-origin`,
+    `unsafe-url`, `same-origin`, `strict-origin`, or
+    `strict-origin-when-cross-origin`. Defaults to
+    `strict-origin-when-cross-origin`.
+  * `cache` string (optional) - can be `""`, `default`, `no-store`, `reload`,
```

Comment:
So is `cache` here.

Reply:
Hm, the [spec](https://fetch.spec.whatwg.org/#requestcache) doesn't require the empty string here. I thought I had put this in here for a reason, but I can't remember what it was any more, so I'll remove it.